.travis.yml

```diff
@@ -22,6 +22,8 @@ matrix:
       env: WP_VERSION=3.7.11
     - php: 5.3
       env: WP_VERSION=3.7.11 DEPLOY_BRANCH=master
+    - php: 7.1
+      env: WP_VERSION=latest SAVE_COMMENTS_DISABLED=1
```

Comment:
That was actually the way I tried it first off, but it makes the PHP 7.1 job very long, whereas there's some parallelism doing it separately (and you get early failure in the first key case) - but it's much of a muchness - I'll put it into the existing job and see what you think.